### PR TITLE
update spanish translations

### PR DESCRIPTION
**Pull request by capitantrueno**

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -55,11 +55,11 @@ msgstr "Terminar proceso"
 #: src/Widgets/Headerbar.vala:30
 #, fuzzy
```
Comment (ryonakano) [inline po/es.po]:
```suggestion
```

You need to remove the `fuzzy` tag, otherwise your translation won't be shown in the app.

File: po/es.po [hunk 2]
```diff
@@ -68,7 +68,7 @@ msgstr "Preferencias"
 #: src/Widgets/Headerbar.vala:56
 #, fuzzy
```
Comment (ryonakano) [inline po/es.po]:
```suggestion
```

Same here.

File: po/es.po [hunk 3]
```diff
@@ -68,7 +68,7 @@ msgstr "Preferencias"
...
 msgid "Show an indicator:"
-msgstr "Mostrar icone de panel"
+msgstr "Mostrar un icono en el panel"
```
Comment (ryonakano) [inline po/es.po]: I don't understand Spanish, but is it intentional that there is no trailing colon that the source string has?

File: po/es.po (hunk 1 above)
Comment (capitantrueno) [reply po/es.po]: Thank you for your suggestion!

File: po/es.po (hunk 3 above)
Comment (capitantrueno) [reply po/es.po]: Thank you for your suggestion!

File: po/es.po (hunk 2 above)
Comment (capitantrueno) [reply po/es.po]: Done!

**Pull request merged**